dist/index.html

```diff
@@ -5,12 +5,12 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Leader Board</title>
-<script defer src="index.bundle.js"></script><script defer src="Functions.bundle.js"></script></head>
+<script defer src="main.bundle.js"></script></head>
 <body>
     <h1 class="heading">Leader Board</h1>
     <h6 class="recent">Recent Scores</h6>
-    <a href="index.html"><button class="referesh">Referesh</button></a>
-    <div class="scores"></div>
+    <button class="referesh">Referesh</button>
+    <div class="scoresDiv scores"></div>
     <h6 class="Add-head">Add your score</h6>
     <form action="">
         <input type="text" placeholder="Your Name" class="Name">
```

Comment:
- [ Optional ] Great job working on this milestone of the application, I think it would be even better if you made these two input fields `required` so that a user is not able to submit blank data.

```HTML
    <input type="text" placeholder="Your Name" class="Name" required>
    <input type="text" placeholder="Your Score" class="Score" required>
```